spec/features/user_views_reviews_spec.rb

```diff
@@ -35,7 +35,7 @@
     visit item_path(@item)
 
     expect(page).to have_selector("ul#review_list li:nth-child(1)", text: @review_3.description)
-    expect(page).to have_selector("ul#review_list li:nth-child(2)", text: @review_2.description)
-    expect(page).to have_selector("ul#review_list li:nth-child(3)", text: @review_1.description)
+    expect(page).to have_selector("ul#review_list li:nth-child(3)", text: @review_2.description)
+    expect(page).to have_selector("ul#review_list li:nth-child(5)", text: @review_1.description)
```

Comment:
If you are looking for a better way to scope your expectations using capybara, take a look at the `within` method:

http://www.rubydoc.info/github/jnicklas/capybara/Capybara%2FSession%3Awithin